lib/ts/utils.ts

```diff
@@ -205,3 +207,49 @@ export function validateTheStructureOfUserInput(
         });
     }
 }
+
+export async function getDataFromFileForServerlessCache<T>(filePath: string): Promise<T | undefined> {
+    try {
+        let dataFromFile = await new Promise<Buffer>((resolve, reject) => {
+            readFile(filePath, (err, data) => {
+                if (err !== undefined && err !== null) {
+                    reject(err);
+                }
+                resolve(data);
+            });
+        });
+        return JSON.parse(dataFromFile.toString());
+    } catch (err) {
+        return undefined;
+    }
+}
+
+export async function storeIntoTempFolderForServerlessCache(filePath: string, data: any) {
+    try {
+        await new Promise(async (resolve, reject) => {
+            writeFile(filePath, JSON.stringify(data), (err) => {
```

Comment:
You are using JSON stringify here. For the API version, you are only giving this a string, will that work?

Reply:
yes